src/test/java/org/folio/linked/data/util/JsonUtilsTest.java

```diff
@@ -0,0 +1,25 @@
+package org.folio.linked.data.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.linked.data.test.TestUtil.loadResourceAsString;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.folio.spring.testing.type.UnitTest;
+import org.junit.jupiter.api.Test;
+
+@UnitTest
+public class JsonUtilsTest {
```

Comment:
minor - "public" not needed for the class and test method.

Reply:
done.